with-jest/jest.config.mjs

```diff
@@ -17,9 +17,8 @@ const config = {
   }),
   testEnvironment: "jsdom",
   transform: {
-    "^.+\\.ts?$": ["ts-jest", { isolatedModules: true, useESM: true }],
-    "^.+\\.tsx?$": ["ts-jest", { useESM: true, tsconfig: { jsx: "react-jsx" } }]
```

Comment:
afaik, this "should" work when running with the example as root. But yeah not sure if it would work from monorepo root :-?